Mathlib/RingTheory/MvPowerSeries/LinearTopology.lean

```diff
@@ -135,6 +136,34 @@ instance [IsLinearTopology R R] [IsLinearTopology Rᵐᵒᵖ R] :
     IsLinearTopology (MvPowerSeries σ R)ᵐᵒᵖ (MvPowerSeries σ R) :=
   IsLinearTopology.mk_of_hasBasis'  _ hasBasis_nhds_zero (fun J _ _ hg ↦ J.mul_mem_right _ _ hg)
 
+theorem isTopologicallyNilpotent_of_constantCoeff
```

Comment:
Is it worth having the `HasEval` consequence, and/or the same for `HasSubst`?

Reply:
I think these should rather come once we need them, because out of all the statements I could write corresponding to your idea I'm not sure which one would be more useful. In any case, it's one rewrite of `hasEval_def` + `isTopologicallyNilpotent_of_constantCoeff` away.

Reply:
OK